nengo_viz/components/slider.py

```diff
@@ -36,13 +40,21 @@ def javascript(self):
                     label=self.label,
                     start_value=[float(x) for x in self.start_value])
         json = self.javascript_config(info)
-        return 'new VIZ.Slider(main, %s);' % json
+        return 'new VIZ.Slider(main, sim, %s);' % json
+
+    def update_client(self, client):
+        while len(self.data) > 0:
+            data = self.data.pop(0)
```

Comment:
As noted in #148, this is O(n^2). I dunno if it really matters here, but regardless, should we do the same optimization as in that PR?